scripts/SplitStockholm.jl

```diff
@@ -14,9 +15,12 @@ function parse_commandline()
             help = "Input file"
             required = true
         "--path", "-p"
-        		help = "Path for the output files [default: execution directory]"
-		        arg_type = String
-    		    default = ""
+            help = "Path for the output files [default: execution directory]"
+            arg_type = String
+            default = ""
+        "--progress"
```

Comment:
Change `"--progress"` for `"--hideprogress"`, in this way the progress id `true` by default.